src/st_widgets.c

```diff
@@ -848,7 +848,7 @@ static void UpdateMonSec(sbe_widget_t *widget)
     int itemcolor =
         (fullitemcount >= totalitems) ? '0' + cr_blue : '0' + CR_GRAY;
 
-    char kill_str[16], item_str[16], secret_str[16];
+    char kill_str[80], item_str[80], secret_str[80];
```

Comment:
Why is this necessary?

Reply:
That'd justify an increasse from 16 to 20, right? 😁

Reply:
It's okay, of course.

Reply:
> That'd justify an increasse from 16 to 20, right? 😁

Yes, we don't have much screen space anyway. However, I think it's okay to have slightly oversized buffers.